README.md

```diff
@@ -9,72 +9,93 @@
-You can create a Workspace with the dependencies and configuration needed to preview the docs locally by executing the following command:
+The most efficient way to begin working on the documentation is by utilizing a development environment manager (DEM) like Daytona. 
+To set up a Workspace with the necessary dependencies and configurations for local preview, run the following command:
 
 ```shell
 daytona create https://github.com/daytonaio/docs.git
 ```
 
-Alternatively, if your local machine has Node.js and Yarn set up, you can clone this repository and execute `yarn install` to install the required dependencies.
+### Manual
+
+Alternatively, if Node.js or Yarn are installed and set up on your local machine, you can clone this repository and install the required dependencies and configurations:
+
+```shell
+git clone https://github.com/daytonaio/docs.git
+```
+
+```shell
+yarn install
+```
 
 ### Useful Commands
 
-This repository uses Yarn to handle project management.
-For convenience, a number of scripts are defined in `package.json` that can be called using Yarn:
-
-| Command                         | Action                                                              |
-| :------------------------------ | :------------------------------------------------------------------ |
-| `yarn run dev`                  | Starts local dev server at `localhost:4321`                         |
-| `yarn run build`                | Builds your production site to `./dist/`                            |
-| `yarn run preview`              | Previews your build locally before deployment                       |
-| `yarn run astro ...`            | Runs CLI commands like `astro add`, `astro check`                   |
-| `yarn run astro -- --help`      | Provides help for using the Astro CLI                               |
-| `yarn run update-cli-reference` | Update the Daytona CLI reference docs using autogenerated CLI docs` |
-
-### Navigating the Docs
-
-On the filesystem, you'll find the content in `./src/content/docs` in the form of `.mdx` files.
-Daytona's documentation is organized into broad topics:
-
-- **Configuration**
-  Information around configuring the main components of Daytona.
-- **About**  
-   Background and conceptual information about Daytona and DEMs.
-- **Installation**  
-   Information around installing Daytona under different environments.
-- **Reference**  
-   Technical and architectural information about Daytona.
-- **Usage**  
-   Practical information on using Daytona.
+This repository utilizes `Yarn` and `npm` to manage project dependencies and custom scripts. Below is a list of commonly used commands, defined in the `package.json`:
+
+| Command (`Yarn`)             | Command (`npm`)              | Action                                                                       |
+| :------------------------- | :----------------------------- | :--------------------------------------------------------------------------- |
```

Comment:
Remove the `npm` commands column since we have `yarn.lock` committed into the repo